src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Public/NLHandler.cs

```diff
@@ -64,7 +65,13 @@ public class NL2FxParameters
         public CheckContextSummary SymbolSummary { get; set; }
 
         // Engine can provide ambient details for NL, such as config and feature flags.
-        public Engine Engine { get; set; }
+        public Engine Engine { get; set; }
+
+        /// <summary>
+        /// Set the locale that expect the output expression to be.
+        /// For example: "vi-VN" or "fr-FR".
+        /// </summary>
+        public CultureInfo ExpressionCultureInfo { get; set; }
```

Comment:
>ExpressionCultureInfo [](http://example.com/codeflow?start=27&length=21)

nit: most places we have named CultureInfo instance as `Locale`.

Reply:
update name